scripts/flavors.erb

```diff
@@ -0,0 +1,235 @@
+require 'fog/aws/models/compute/flavor'
+
+module Fog
```

Comment:
It would probably be ok to leave this file out. I don't think we really need to build objects in order to do the formatting stuff. I think just filling in the templates directly from the data would probably make it easier to maintain/debug as needed. Does that make sense?

Reply:
Let me rephrase what I think you're trying to say, to make sure I understand. You're saying to not include the .erb template, but to write out a static file directly from the sorted hashed set?

In my mind, I was trying to introduce as little change as possible, probably at some point including a build step that generates the model file before the actual tests and packaging.